### PR TITLE
bug fixes

### PR DESCRIPTION
**Pull request by flacoman91**
- changed require statement to correctly load depenencies
- changing changedAt to updatedAt since that's what is saved in mongoDB

### CODE REVIEW

File: diffHistory.js [hunk 1]
```diff
@@ -130,7 +130,7 @@ var getHistories = function (modelName, id, exapndableFields, callback) {
             var comment = "modified " + changedFields.concat(changedValues).join(", ");
             return mapCallback(null, {
                 changedBy: history.user,
-                changedAt: history.created_at,
+                updatedAt: history.updatedAt,
```
Comment (mimani) [inline diffHistory.js]: Should we use createdAt here?
https://github.com/mimani/mongoose-diff-history/issues/10

Comment (flacoman91) [inline diffHistory.js]: Sorry, I missed that. I think it's a good idea to have both values in there.

File: diffHistory.js [hunk 2]
```diff
@@ -130,7 +130,8 @@ var getHistories = function (modelName, id, expandableFields, callback) {
             var comment = "modified " + changedFields.concat(changedValues).join(", ");
             return mapCallback(null, {
                 changedBy: history.user,
-                changedAt: history.created_at,
```
Comment (mimani) [inline diffHistory.js]: Can you please keep changedAt as well, as someone might be using it, as following - 

changedAt: history.createdAt,

Comment (flacoman91) [inline diffHistory.js]: sure. no problem.

**Pull request merged**